### PR TITLE
units/tregxx0001: test for registrar module and xavp_rcd_mask

### PR DESCRIPTION
**Pull request by linuxmaniac**
depends on functionality to be added at https://github.com/kamailio/kamailio/pull/2168

### CODE REVIEW

**Review by giavac — APPROVED** [review]
Thanks Victor. I'm approving this and perhaps we can merge after the related PR for kamailio is merged?
Otherwise we could add this to the excluded tests and merge immediately.

**Comment by linuxmaniac** [comment]
We can wait until the PR is merged. No rush at all.

**Pull request merged**
